### DISCUSSION TITLE
Blinking Icon

### DISCUSSION THREAD

**Discussion by djalexz** [Zefau/ioBroker.jarvis#2089] [Q&A] [answered]
Hallo zusammen,

nicht wirklich ein Bugreport, eher eine Frage. Ich bekomme es nicht hin.
Aktuellste Jarvis Beta.

Ich habe einen Datenpunkt, der im Normalfall "kein Fehler" anzeigt.
Da im Fehlerfall über 50 versch. Variablen angezeigt werden können, würde ich gerne ein blinkendes Icon haben, wenn "nicht" "kein Fehler" im Datenpunkt steht.

Mit `{"!=kein Fehler":"akar-icons:info blink"}` funktioniert es leider nicht.

Bitte um Hilfe.

Vielen Dank

LG Alex

**Comment by mcuiobroker** [comment, marked as answer]
`{"default":"akar-icons:info blink","kein Fehler":""}`

**Comment by djalexz** [comment]
Klasse, ich danke dir sehr.
Hab mir gefühlte 5 Stunden die Finger wund gesucht...
Sollte evtl. mit in die Doku aufgenommen werden.

LG Alex

**Reply by mcuiobroker** [reply]
Es steht in der Doku.
https://mcuiobroker.gitbook.io/jarvis-infos/jarvis/besonderheiten-v3/geraet/eingabewerte-fuer-die-geraete-dp-state-felder

Aber wo hat es dir gefehlt? Wo hast du gesucht?

**Reply by djalexz** [reply]
Ich hab genau da gesucht, aber irgendwie total übersehen.
Danke dir nochmals. Ich war einfach blind 🙈
